lib/dat-worker-pool/runner.rb

```diff
@@ -57,19 +57,18 @@ def decrement_worker_waiting
       @workers_waiting.decrement
     end
 
+    def spawn_worker(worker)
```

Comment:
@jcredding we discussed renaming these to add/remove worker since the spawn/despawn doesn't necessarily apply anymore.